ccd-definition/CaseField/CareSupervision/caseField.json

```diff
@@ -1756,15 +1756,6 @@
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "isOutsourcingLA",
-    "Label": "Are you a solicitor creating a case on behalf of a local authority?",
-    "FieldType": "YesOrNo",
-    "SecurityClassification": "Public",
-    "Searchable": "N"
-  },
```

Comment:
would perhaps recommend keeping this one, I imagine it's cleared on most peoples cases but just incase it's not we can leave it in, perhaps move it to the DEPRECATED file instead